cfgov/v1/tests/wagtail/test_molecules.py

```diff
@@ -0,0 +1,206 @@
+from django.test import TestCase
+from django.test import Client
+
+from scripts import _atomic_helpers as atomic
+from v1.models.landing_page import LandingPage
+from v1.models.sublanding_page import SublandingPage
+from v1.models.learn_page import LearnPage, DocumentDetailPage
+from v1.models.browse_page import BrowsePage
+from v1.models.browse_filterable_page import BrowseFilterablePage
+from v1.models.sublanding_filterable_page import SublandingFilterablePage
+from wagtail.wagtailcore.blocks import StreamValue
+from helpers import publish_page
+
+c = Client()
```

Comment:
I would agree with renaming the variable... especially if you're looking at the file for a single test, it may be non-obvious at first what `c` is.